src/java/us/kbase/templates/python_server.vm.properties

```diff
@@ -129,11 +129,10 @@ class JSONRPCServiceCustom(JSONRPCService):
             # Exception was raised inside the method.
             newerr = JSONServerError()
             newerr.trace = traceback.format_exc()
-            if isinstance(e.message, str):
-                newerr.data = e.message
+            if len(e.args) == 1:
```

Comment:
I think this is fine but hacky. I don't know if stdout will log properly in this class, but we could also just do something like:
```py
traceback.print_exc()
print('*' * 80)
print(str(e))
print('*' * 80)
raise JSONServerError()
```